test/unit/presenters/document_export_presenter_test.rb

```diff
@@ -159,6 +159,17 @@ class DocumentExportPresenterTest < ActiveSupport::TestCase
                  result.dig(:editions, 0, :authors, 0)
   end
 
+  test "includes contacts" do
+    contact = create(:contact)
+    edition = create(:news_article,
+                     body: "[Contact:#{contact.id}]",
+                     depended_upon_contacts: [contact])
+
+    result = DocumentExportPresenter.new(edition.document).as_json
+    assert_equal ([{ id: contact.id, content_id: contact.content_id }]), result.dig(:editions, 0, :contacts)
+    assert_nil result.dig(:editions, 0, :depended_upon_contacts)
```

Comment:
I'm not quite sure how this this is `nil` as we don't seem to be making any effort to remove it 🤔

Reply:
We don't need to remove anything as it's not automatically populated. I've removed this test condition as I don't think there's a particular reason we're bothered about it's presence or not.